packages/bun-usockets/src/crypto/openssl.c

```diff
@@ -220,6 +220,24 @@ void us_internal_on_ssl_handshake(
   context->handshake_data = custom_data;
 }
 
+struct us_internal_ssl_socket_t *
+us_internal_ssl_socket_close(struct us_internal_ssl_socket_t *s, int code,
+                             void *reason) {
+  struct us_internal_ssl_socket_context_t *context =
+      (struct us_internal_ssl_socket_context_t *)us_socket_context(0, &s->s);
```

Comment:
Can you write a long comment explaining why we do this? It’s not clear from this code why we should go through handshaking when we’re about to close the connection. If the goal is to clear the TLS state maybe there’s something we can do more directly.

Reply:
I dont think is is necessary anymore, will look and fix

Reply:
actually is necessary added a comment